homeassistant/components/media_player/channels.py

```diff
@@ -33,8 +33,6 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
-    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
Don't remove this if you still expect it in the config. All config options must be validated in the config schema.